lib/docker.js

```diff
@@ -141,21 +153,21 @@ function createDockerFile() {
     }
 
     if (opts.uid !== 0) {
-        contents += 'RUN groupadd -g ' + opts.gid + ' -r rungroup && ' +
-        'useradd -m -r -g rungroup -u ' + opts.uid + ' runuser\n' +
+        contents += 'RUN groupadd -r rungroup && ' +
```

Comment:
Ups, sorry, that's a workaround for testing on my mac, I forgot I didn't remove it when made a second commit.